site/dat/docs/JMeter.md

```diff
@@ -166,6 +166,7 @@ scenarios:
       delimiter: ';'  # CSV delimiter, auto-detected by default
       quoted: false  # allow quoted data
       loop: true  # loop over in case of end-of-file reached
+      stop-on-eof: false  # stop test execution if end-of-file is reached (requires `loop` to be `false`)
```

Comment:
if it is mutually exclusive with 'loop", why not have some "special" value for "loop" that will tell stop instead of looping?